domains/transfer/ClaimAutomation/Worker.IntegrationTests/ClaimAutomationApplicationFactory.cs

```diff
@@ -29,7 +29,7 @@ public class ClaimAutomationApplicationFactory : WebApplicationFactory<Program>,
 
     public Task InitializeAsync() => testContainer.StartAsync();
 
-    Task IAsyncLifetime.DisposeAsync() => testContainer.DisposeAsync().AsTask();
+    async Task IAsyncLifetime.DisposeAsync() => await testContainer.DisposeAsync();
```

Comment:
Why is this better?

Reply:
Not necessarily better, but the normal patter is to await an async method. So why did this return .asTask() instead?